viite-UI/src/utils/LinkValues.js

```diff
@@ -0,0 +1,79 @@
+(function (root) {
```

Comment:
this is really nice addition

Reply:
Just remember to add these to integration-tests.html as well.